deploy/016_deploy_zksync_spokepool.ts

```diff
@@ -31,6 +32,14 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // When CCTP is available, set the new addresses for cctpTokenMessenger and USDC to enable.
   const usdcAddress =
     zkUSDCBridge === ZERO_ADDRESS && cctpTokenMessenger === ZERO_ADDRESS ? ZERO_ADDRESS : USDC[spokeChainId];
+  if (usdcAddress !== ZERO_ADDRESS) {
+    const cctpTokenMessengerDefined = cctpTokenMessenger !== ZERO_ADDRESS;
+    const zkUSDCBridgeDefined = zkUSDCBridge !== ZERO_ADDRESS;
+    assert(
+      cctpTokenMessengerDefined || zkUSDCBridgeDefined,
```

Comment:
We're cool allowing both to be defined?

If we want to make sure _exactly one_ is defined, we should change the assert to `cctpTokenMessengerDefined !== zkUSDCBridgeDefined`.